diff --git a/node_modules/@1inch/fusion-sdk/dist/cjs/constants.js b/node_modules/@1inch/fusion-sdk/dist/cjs/constants.js
index 198896c..793af4e 100644
--- a/node_modules/@1inch/fusion-sdk/dist/cjs/constants.js
+++ b/node_modules/@1inch/fusion-sdk/dist/cjs/constants.js
@@ -17,6 +17,8 @@ var NetworkEnum;
     NetworkEnum[NetworkEnum["LINEA"] = 59144] = "LINEA";
     NetworkEnum[NetworkEnum["SONIC"] = 146] = "SONIC";
     NetworkEnum[NetworkEnum["UNICHAIN"] = 130] = "UNICHAIN";
+    NetworkEnum[NetworkEnum["SEPOLIA"] = 11155111] = "SEPOLIA";
+    NetworkEnum[NetworkEnum["NILE"] = 3448148188] = "NILE"; 
 })(NetworkEnum = exports.NetworkEnum || (exports.NetworkEnum = {}));
 exports.ONE_INCH_LIMIT_ORDER_V4 = '0x111111125421ca6dc452d289314280a0f8842a65';
 exports.UINT_160_MAX = (1n << 160n) - 1n;
diff --git a/node_modules/@1inch/fusion-sdk/dist/esm/constants.js b/node_modules/@1inch/fusion-sdk/dist/esm/constants.js
index ca6c4da..ae5f0cd 100644
--- a/node_modules/@1inch/fusion-sdk/dist/esm/constants.js
+++ b/node_modules/@1inch/fusion-sdk/dist/esm/constants.js
@@ -14,6 +14,8 @@ export var NetworkEnum;
     NetworkEnum[NetworkEnum["LINEA"] = 59144] = "LINEA";
     NetworkEnum[NetworkEnum["SONIC"] = 146] = "SONIC";
     NetworkEnum[NetworkEnum["UNICHAIN"] = 130] = "UNICHAIN";
+    NetworkEnum[NetworkEnum["SEPOLIA"] = 11155111] = "SEPOLIA";
+    NetworkEnum[NetworkEnum["NILE"] = 3448148188] = "NILE"; 
 })(NetworkEnum || (NetworkEnum = {}));
 export const ONE_INCH_LIMIT_ORDER_V4 = '0x111111125421ca6dc452d289314280a0f8842a65';
 export const UINT_160_MAX = (1n << 160n) - 1n;
diff --git a/node_modules/@1inch/fusion-sdk/dist/types/src/constants.d.ts b/node_modules/@1inch/fusion-sdk/dist/types/src/constants.d.ts
index edfe59d..c08b741 100644
--- a/node_modules/@1inch/fusion-sdk/dist/types/src/constants.d.ts
+++ b/node_modules/@1inch/fusion-sdk/dist/types/src/constants.d.ts
@@ -1,5 +1,7 @@
 export declare const ZX = "0x";
 export declare enum NetworkEnum {
+    SEPOLIA = 11155111,
+    NILE = 3448148188,
     ETHEREUM = 1,
     POLYGON = 137,
     ZKSYNC = 324,
